spec/support/oauth_helper.rb

```diff
@@ -4,9 +4,9 @@ def login_with_oauth(user = create(:user))
     visit "/auth/githubteammember"
   end
 
-  def setup_mock_auth(email = "test@example.com")
+  def setup_mock_auth(email = "test@example.com", overrides = {})
```

Comment:
I added them to test unsuccessful auth, but maybe I could change it to an `options` argument that handles something like `team_member?: false`?